javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,81 @@
+// selects the all HTML element with the class "text-content span" and assigns it to the variable "$textSpans" using JavaScript.
+// IMPORTANT when using querySelectorAll method we are dealing with array type objects
+const $textSpans = document.querySelectorAll('.text-content span');
+console.log($textSpans);
```

Comment:
Remove all `console.log`s before submitting! They are only for testing / debugging purposes. Unless we specifically ask for them in the instructions.